src/main/java/com/twilio/oai/resolver/common/CodegenParameterDataTypeResolver.java

```diff
@@ -10,6 +10,9 @@ public class CodegenParameterDataTypeResolver extends Resolver<CodegenParameter>
     @Setter
     private IConventionMapper mapper;
 
+    public CodegenParameterDataTypeResolver(IConventionMapper mapper) {
+        this.mapper = mapper;
```

Comment:
Can we make `mapper` final and remove the setter?

Reply:
addressed

Reply:
nit: Can use `@RequiredArgsConstructor`